### ISSUE TITLE
Adjust timings via API

### ISSUE THREAD

**Issue by kasperdyrvig** [AntonyCorbett/OnlyT#138]
Most of the time we control the timer via API (the OnlyT remote app). It would be great if the app/api had the ability to adjust timings!

**Comment by LvDaniel** [comment]
I agree, this would be a nice to have option

**Comment by LvDaniel** [comment]
This gives the option to chairman to adjust the timer when there are some announcements to make and letters to be read at the end of the meeting.

**Comment by AntonyCorbett** [comment]
@LvDaniel I would suggest the chairman would normally liaise with the AV desk on this and the OnlyT operator would adjust the timings as necessary. I think this feature request relates to the situation where OnlyT is controlled by a remote device via the web API. And in these circumstances, @kasperdyrvig would like to be able to modify the timings on the device rather than having to go over to the PC to do it.

**Comment by LvDaniel** [comment]
@AntonyCorbett you're right, normally the chairman should communicate upfront any changes.
But if we entrust someone from the audience the responsability to operate the timing, will be a nice option to have instead of going to the AV desk and ask there to do the necessary changes.

**Comment by jnbanda** [comment]
@AntonyCorbett Is this enhancement still on the roadmap? Any plans to include this in the API?

**Comment by AntonyCorbett** [comment]
> I submitted a PR for the API changes: [#576](https://github.com/AntonyCorbett/OnlyT/pull/576)

Thank you - I'll review asap

**Comment by AntonyCorbett** [comment]
Available now in pre-release (2.4.0.17+)
